### PR TITLE
remove requirement for ndep from aquaplanet cases

### PR DESCRIPTION
**Pull request by jedwards4b**
Removes the requirement for an ndep_srf file for aquaplanet cases.

### CODE REVIEW

File: bld/namelist_files/namelist_defaults_cam.xml [hunk 1]
```diff
@@ -3157,8 +3157,8 @@
 <mpas_len_disp hgrid="mpasa60">  60000.0D0</mpas_len_disp>
 <mpas_len_disp hgrid="mpasa30">  30000.0D0</mpas_len_disp>
 <mpas_len_disp hgrid="mpasa15">  15000.0D0</mpas_len_disp>
-<mpas_len_disp hgrid="mpasa7.5">  7500.0D0</mpas_len_disp>
-<mpas_len_disp hgrid="mpasa3.75"> 3750.0D0</mpas_len_disp>
+<mpas_len_disp hgrid="mpasa7p5">  7500.0D0</mpas_len_disp>
+<mpas_len_disp hgrid="mpasa3p75"> 3750.0D0</mpas_len_disp>
```
Comment (gdicker1) [inline bld/namelist_files/namelist_defaults_cam.xml:3161]: Smaller ask: @jedwards4b could you extend this change for the `mpas_dt` section a few lines above?

**Review by gdicker1 — CHANGES_REQUESTED** [review]
I may need guidance on how to use this, but I couldn't get a QPC6 case to run with this change.

I cloned the develop branch of EarthWorks, checked out externals, then merged this into `components/cam` and created a QPC6 case as I normally do:
```
./cime/scripts/create_newcase --compset QPC6 --res mpasa120_mpasa120 --project UCSU0085 --driver nuopc --case QPC6.mpasa120.CAMPR6 --run-unsupported --input-dir /glade/p/univ/ucsu0085/inputdata2/
```

Once setup, built, and submitted (with no extra steps) this case fails in initialization from within [cam/src/chemistry/mozart/mo_drydep.F90 line 382](https://github.com/EarthWorksOrg/CAM/blob/884ae44068d9afa8fec05f6101985f5aa633ac6b/src/chemistry/mozart/mo_drydep.F90#L382) since the `mapping` array from `shr_drydep_mod` isn't allocated.

Could you shed any light on whether my setup is wrong or how to fix this error? @jedwards4b

**Comment by jedwards4b** [comment]
I thought that I had tested this change adequately last week, but I ran into the same problem you are finding when I tried this morning.  Sorry, I'll repackage the non ndep related changes this afternoon so that you can get those at least.

**Comment by gdicker1** [comment]
Since there has been no movement on this in a while, I'll close for now. We may have even gotten the changes through another update!

Feel free to open and refresh this if it still applies!

**Pull request closed without merging**